exercises/simple-linked-list/linked_list_test.exs

```diff
@@ -0,0 +1,155 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("linked_list.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule LinkedListTest do
+  use ExUnit.Case
+
+  test "length/1 of new list" do
+    list = LinkedList.new()
+    assert LinkedList.length(list) == 0
+  end
+
+  @tag :pending
+  test "empty?/1 of new list" do
+    list = LinkedList.new()
+    assert LinkedList.empty?(list)
+  end
+
+  @tag :pending
+  test "length/1 of list of 1 datum" do
+    list = LinkedList.new() |> LinkedList.push(10)
+    assert LinkedList.length(list) == 1
+  end
+
+  @tag :pending
+  test "empty?/1 of list of 1 datum" do
+    list = LinkedList.new() |> LinkedList.push(20)
+    refute LinkedList.empty?(list)
+  end
+
+  @tag :pending
+  test "peek/1 of list of 1 datum" do
+    list = LinkedList.new() |> LinkedList.push(20)
+    assert LinkedList.peek(list) == {:ok, 20}
+    assert LinkedList.length(list) == 0
```

Comment:
I don't think it did run... I've been noticing some odd occasions where a push doesn't trigger a CI run in GitHub recently, but only with open source projects. I'm not sure where the fault lies, but I've reported it to GitHub. Hopefully they can figure it out!

I first noticed it during the DDoS attack earlier and figured it was related to that, but it's still happening, so hopefully it is just a quick thing they can fix.